src/astarte-client/models/Block/CustomBlock.ts

```diff
@@ -0,0 +1,40 @@
+/*
+   This file is part of Astarte.
+
+   Copyright 2020 Ispirata Srl
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import type { AstarteBlockType, AstarteCustomBlockDTO } from '../../types';
```

Comment:
`../..` in imports doesn't look too good. We can alias the astarte directory for `astarte/types` or alias types directly

Reply:
That's a good point. I was thinking about postponing this alias to when we extract the Astarte Client into a package, since these types are "private" and only used inside Astarte Client and I didn't want to expose this alias to the js app together with the `astarte-client` alias.